### PR TITLE
have a $service_name parameter for all prometheus-exporters

### PR DESCRIPTION
**Pull request by unki**
Currently there is a mixture of exporters having a `$service_name` parameter or not.
I've tried to make them all to look the same know.

### CODE REVIEW

File: manifests/consul_exporter.pp [hunk 1]
```diff
@@ -63,6 +65,7 @@
   String $log_level,
   String $package_ensure,
   String $package_name,
+  String $service_name,
```
Comment (bastelfreak) [inline manifests/consul_exporter.pp]: for all `$service_name` parameters please use `String[1]` to prohibit empty empty strings

Comment (unki) [reply manifests/consul_exporter.pp]: Sure, will adapt the PR. Shall I change it for `$package_name` too?

As far as I remember I always have just copied the package-name-line as base and adapted it for the service. So there might be a mixture of `String` and `String[1]` there too.

Comment (bastelfreak) [reply manifests/consul_exporter.pp]: yes please :)

**Comment by bastelfreak** [comment]
@unki please have a look at the failing travis jobs

**Comment by unki** [comment]
> @unki please have a look at the failing travis jobs

ready to go now

**Comment by bastelfreak** [comment]
thanks for the work!

**Comment by elbandi** [comment]
looks like, not all exporters have this $service_name as daemon name

**Pull request merged**